config/demo-conf.toml

```diff
@@ -4,6 +4,7 @@ AdminHost = "127.0.0.1" # optional, default to localhost
 AdminPort = 7778 
 #LogDir = "/var/log/lmstfy"
 LogLevel = "info"
+#LogFormat = "text" # If need json format logs, change it to `json`
```

Comment:
# If need json format logs, change it to `json` => # use LogFormat="json" if wants to print the log with json format